src/main/java/formflow/library/ScreenController.java

```diff
@@ -63,6 +64,7 @@ public class ScreenController extends FormFlowController {
 
   public static final String FLOW = "/flow";
   public static final String FLOW_SCREEN_PATH = "{flow}/{screen}";
+  private final String DELETE_CONFIRMATION_REDIRECT_UUID = "deleteConfirmationRedirectUuid";
```

Comment:
I'm going to take this out. It was for an intermediate impl attempt that we didn't end up using.